ChangeLog

```diff
@@ -494,6 +494,10 @@ Changes
    * Ciphersuites based on 3DES now have the lowest priority by default when
      they are enabled.
 
+Features
```

Comment:
Please add a file in `ChangeLog.d` instead of modifying `ChangeLog` directly. We changed this to avoid recurring add/add conflicts in `ChangeLog`.